fusion/sample_production/group_vars/all

```diff
@@ -1,8 +1,8 @@
 # read from OS vars
-api_client: "{{ ansible_env.API_CLIENT }}"
+issuer_id: "{{ ansible_env.ISSUER_ID }}"
 priv_key_file: "{{ ansible_env.PRIV_KEY_FILE }}"
```

Comment:
Since you've changed `api_client` to `issuer_id`, let's change `priv_key_file` to `private_key_file` too (nice to have thing).